Servers/Atlas/.arc/submit.py

```diff
@@ -104,17 +104,15 @@
 
 touch initial_time
 
-SubmitDir={pwd}
-
 mkdir -p $MOLPRO_SCRDIR
 cd $MOLPRO_SCRDIR
 
-cp "$SubmitDir/input.in" .
+cp "$SUBMIT_DIR/input.in" .
 
 /Local/ce_dana/molpro-mpp-2022.2.3/bin/molpro -n {cpus} -t 1 input.in
 
-cp input.* "$SubmitDir/"
-cp geometry*.* "$SubmitDir/"
+cp input.* "$SUBMIT_DIR/"
+cp geometry*.* "$SUBMIT_DIR/"
```

Comment:
Does molpro always create a geometry file?

In the Zeus and Azure scripts, I put in an if statement to copy if it exists

`if [ ! -f geometry.* ]; then
	echo "Geometry files don't exist at this time"
else
	cp geometry.* ""$PBS_O_WORKDIR"/"
fi`

Maybe it's good to have that?